src/tests/data/create-admin-user.yaml

```diff
@@ -0,0 +1,5 @@
+---
+Username: test@gmail.com
+Roles:
+- Administrator
+State: added
```

Comment:
Is `state` really an input? Can this be omitted or does the api actually force this to be present?

Reply:
This was present in request body of the api call made from UI